prow/resultstore/payload.go

```diff
@@ -64,15 +64,17 @@ func (p *Payload) invocationID() *resultstore.Invocation_Id {
 
 func (p *Payload) invocationStatusAttributes() *resultstore.StatusAttributes {
 	status := resultstore.Status_TOOL_FAILED
```

Comment:
Nit: Do these resultstore.Status_* enums map 1:1 to the job status states? If it's not a clean 1:1 (e.g. there are more fields in one or the other), it's probably worth documenting the discrepancy and what we're leaving out.